Moodlet/app/src/main/java/com/cmput3owo1/moodlet/models/EmotionalState.java

```diff
@@ -4,19 +4,36 @@
 import android.graphics.Color;
 
 public enum EmotionalState {
-    HAPPY(),
-    SAD(),
-    ANGRY(),
-    EXCITED(),
-    CONFUSED(),
-    SCARED(),
-    JEALOUS();
-
-    private Color color;
+    HAPPY("Happy",Color.YELLOW),
+    SAD("Sad",Color.CYAN),
+    ANGRY("Angry",Color.RED),
+    EXCITED("Excited",Color.rgb(255,128,0)),
+    CONFUSED("Confused",Color.rgb(178,102,255)),
+    SCARED("Scared",Color.rgb(204,204,255)),
+    JEALOUS("Jealous", Color.rgb(102,255,178));
```

Comment:
These colors should be changed to constants and placed in the `colors.xml` file in the `/app/src/main/res/values` directory.